app/domain/authentication/authn_jwt/input_validation/validate_claim_name.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Validate the claim name value
+      ValidateClaimName ||= CommandClass.new(
+        dependencies: {
+          regexp_class: Regexp,
+          deny_claims_list_value: [],
+          logger: Rails.logger
+        },
+        inputs: %i[claim_name]
+      ) do
+        def call
+          @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatingClaimName.new(@claim_name))
+          validate_claim_name_exists
```

Comment:
If we use regex to validate the claim? It will also validate its not empty ?
or we want different error for empty claims?

Reply:
i prefer a different error because you should not call this command class with nil or empty value